Nucleon/Nucleon.json

```diff
@@ -0,0 +1,209 @@
+{
+    "type": "DashboardV2SyncDefinition",
+    "name": "Nucleon",
+    "description": "Nucleon learns about the external threat landscape in order to determine how threats can impact your security posture. Nucleon delivers notifications for pre-attack adversary behaviour, empowering you to truly protect your network. As part of its core functions, the Nucleon polymorphic sensors identify impending threats before impact, automating these early-warnings effectively. With our active threat feeds and robust client integrations, these all equal early threat detection and visibility, and increased protection. An additional set of these polymorphic sensors sit closer to the gateways of your network, collecting data about sensitive and vulnerable access points that are unique to your network specifically. They expand your attack surface in order to lure in and potentially trap those bad actors. We provide only actionable alerts, actively guiding you to block the attack source with the help of your existing defense systems. From the Nucleon dashboard, you can access threat intelligence assessments, threat alerts, insights and remediation guidance that contribute to a strengthened security posture. The process is described in detail. For any questions or concerns please contact us through your Slack channel or at Support@nucleon.sh",
+    "title": "Nucleon",
+    "rootPanel": null,
+    "theme": "Dark",
+    "topologyLabelMap": {
+        "data": {}
+    },
+    "refreshInterval": 0,
+    "timeRange": {
+        "type": "BeginBoundedTimeRange",
+        "from": {
+            "type": "RelativeTimeRangeBoundary",
+            "relativeTime": "-15m"
```

Comment:
last 15 minutes seems very small normally our dashboards have last 60 min as default just checking whether 15 min is intentional and not by mistake.

Reply:
I changed it to -60m

Reply:
updated the screenshot to size 1903x711